docs/documentation/make-first-prototype/add-questions.md

```diff
@@ -0,0 +1,52 @@
+# Add questions to your question pages
+
+Copy the code from the [GOV.UK Design System](https://design-system.service.gov.uk/) to add page elements like radios and text inputs.
+
+## Add radios to question 1
+
+1. Go to the [stacked radios section of the Design System](https://design-system.service.gov.uk/components/radios/#stacked-radios).
+2. Select **HTML**, then select **Copy code**.
+3. Open `juggling-balls.html` in your `app/views` folder.
+4. Replace the 2 example `<p>...</p>` paragraphs with the HTML code you copied.
+5. Delete the whole `<h1>` tag that has the text “Heading or question goes here” - it should be line 16.
```

Comment:
In the create pages step, we've [asked the user to update the H1 with the text 'How many balls can you juggle?'](https://github.com/alphagov/govuk-prototype-kit/pull/805/files#diff-989471efd2530184cab03b0727bc9e5dR52) so when they get to this step there will no longer be an h1 that has the text “Heading or question goes here”.

It also seems strange that we ask them to update an `<h1>` that they then delete a few steps later?

Reply:
I think this is correct - because there are *two* `<h1>'s. One is the original version from the template (which is the one we ask the user to edit), the other is the one that comes with the copy-pasted Design System HTML code (which as the 'Heading or question goes here' text).

Reply:
I don't think so – the HTML they copy from the example in the Design System will include the text 'Where do you live?', which is the `<h1>` that's referenced in step 6. (It'll also likely have ended up a lot further down than line 16, because it'd be pasted into the block starting on line 20.)

Reply:
Ah yes, I see what you mean. I've fixed this for both questions in the latest two commits, so we're asking the user to cut and paste the original `<h1>` into the code they copied from the Design System. (So they're definitely left with only one `<h1>`)

Hopefully this helps with the flow too, because we're no longer asking the user to delete the original `<h1>` they edited - just to cut and paste it somewhere else.